frontend/src/util/Hook.tsx

```diff
@@ -1,7 +1,9 @@
 import { useState, useEffect } from 'react';
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
 import { Player, Submission } from '../api/Game';
+import { AppDispatch, RootState } from '../redux/Store';
 
-const useBestSubmission = (player?: Player) => {
+export const useBestSubmission = (player?: Player) => {
```

Comment:
Possibly - I might lean towards keeping it as is though, since it seems more like a custom hook that takes in a certain player and returns that player's best submission, rather than a single global state used on multiple pages